docs/api.md

```diff
@@ -174,20 +174,22 @@ document
   .addEventListener('monetizationstop', stopEventHandler)
 ```
 
-#### Event Object
+#### Event object
 
 | Field | Value | Details |
 | ------ | ----- | ------ |
 | paymentPointer | String | The payment account URL for the Web Monetization `meta` tag at the time when payment has stopped. |
-| requestId | String | The session ID/Monetization ID (UUID v4) [generated by the browser](/docs/explainer#flow) for the Web Monetization `meta` tag at the time when payment has stopped. |
+| requestId | String | The session ID/monetization ID (UUID v4) [generated by the browser](./explainer#flow) for the Web Monetization `meta` tag at the time when payment has stopped. |
+| finalized | Boolean | When `true`, the monetization tag has been removed or the `paymentPointer` changed. No more events with this `requestId` are expected.
```

Comment:
If this is pulling in the `finalized` feature then we'll have to wait until that is published through the extension before we merge this. So I think it's probably best to take out the finalized stuff in this version and let Niq rebase it on top of yours once this gets merged

Reply:
Removed.